### ISSUE TITLE
Compile error... missing argument passthrough here

### ISSUE THREAD

**Issue by OwenKnight-FLINTpro** [mohaps/lrucache11#22]
https://github.com/mohaps/lrucache11/blob/ab288eee4d7148a93e467eb2782bad852b95da6b/LRUCache11.hpp#L161C14-L161C14

**Comment by runner-ccl** [comment]
` Value get(const Key& k) {
    return getCopy(k);
  }`

**Comment by OwenKnight-FLINTpro** [comment]
Yep, have forked locally and fixed it. This project looks dead though, will move on.

**Comment by mohaps** [comment]
Owen,

Do you wish to take over the project. Be glad to do so.

I haven't worked on this for a while

-- Saurav
-- 
"quidquid latine dictum sit, altum sonatur."
--anonymous
-- 

SAURAV MOHAPATRA
*Author, Artist and Bona Fide Geek*
*www <https://sauravmohapatra.com> | **facebook*
<https://fb.me/sauravmohapatraofficial> | *youtube
<https://youtube.com/mohaps>*| *vimeo <https://vimeo.com/mohaps>* | *instagram
<https://instagram.com/flyinghogmonkey>* | *twitter*
<https://twitter.com/mohaps> | *imdb
<https://www.imdb.com/name/nm11131286/>*
